vars/buildManifest.groovy

```diff
@@ -16,8 +16,16 @@
  @param args.snapshot <optional> - Boolean value. Defaults to null.
  @param args.lock <optional> - Generate a stable reference manifest. Defaults to null.
  @param args.continueOnError <optional> - Do not fail the distribution build on any plugin component failure. Defaults to null
+ @param args.incremental <optional> - Boolean value to enable incremental build.
  */
 void call(Map args = [:]) {
+    boolean incremental_enabled = args.incremental != null && args.incremental
+
+    if (incremental_enabled) {
+        echo("Incremental build enabled! Retrieving previous build library.")
+        retrievePreviousBuild(args)
```

Comment:
The arg `previousBuildId` is not passed here.
Is the default `latest` here?

Reply:
It will be passed from the arguments of this class right? The default latest is assigned when we declare the parameter.

Reply:
Since the default is pointing to latest we should be good but if not, whoever calls this lib needs to provide `args.previousBuildId`. Not sure what is the best approach. Maybe mentioning in the comments above?